src/SessionManager.ts

```diff
@@ -117,19 +145,39 @@ export class SessionManager extends EventEmitter {
     return session;
   }
 
+  /**
+   * @param sessionId A unique session identifier.
+   * @returns true if a session associated to the given ID exists.
+   */
   async hasSession(sessionId: string): Promise<boolean> {
     await this.init();
     return (await this.clientAuthn.getSessionInfo(sessionId)) !== undefined;
   }
 
+  /**
+   * Registers a callback to be called when a session is logged in.
+   *
+   * @param callback a function executed when a session logs in, with the session as a parameter.
+   */
   onSessionLogin(callback: (session: Session) => unknown): void {
     this.on("sessionLogin", callback);
   }
 
+  /**
+   * Registers a callback to be called when a session is logged out.
+   *
+   * @param callback a function executed when a session logs out, with the session as a parameter.
+   */
   onSessionLogout(callback: (session: Session) => unknown): void {
-    this.on("sessionLogin", callback);
+    this.on("sessionLogout", callback);
   }
 
+  /**
+   * Removes a session from the pool managed by the manager.
+   * This does not log out the session, but it does remove references to it, so after this call the session will become unreachable.
```

Comment:
Why would you want to do this? (The answer might be a relevant addition to this docblock.)

Reply:
This is useful after a user has logged out, otherwise your server will have an ever-growing list of active sessions. But indeed, that sounds like useful context ^^

Reply:
Documented in 9ab620d1620dd73612b5dfa336fdcb71ac2e8925